indico/modules/events/abstracts/controllers/management.py

```diff
@@ -55,7 +55,7 @@ class RHManageAbstractsBase(RHConferenceModifBase):
     """Base class for all abstracts management RHs"""
 
     CSRF_ENABLED = True
-    EVENT_FEATURE = 'abstracts'
+    EVENT_FEATURE = None
```

Comment:
:warning: only override this in the specific RH for the page that should always be accessible.